app/scripts/lib/setupSentry.js

```diff
@@ -28,72 +29,269 @@ export const ERROR_URL_ALLOWLIST = {
 // debugging, and they do not contain any identifiable information.
 export const SENTRY_BACKGROUND_STATE = {
   AccountTracker: {
+    accounts: false,
     currentBlockGasLimit: true,
   },
+  AddressBookController: {
+    addressBook: false,
+  },
   AlertController: {
     alertEnabledness: true,
+    unconnectedAccountAlertShownOrigins: false,
+    web3ShimUsageOrigins: false,
+  },
+  AnnouncementController: {
+    announcements: true,
   },
   AppMetadataController: {
     currentAppVersion: true,
+    currentMigrationVersion: true,
     previousAppVersion: true,
     previousMigrationVersion: true,
-    currentMigrationVersion: true,
+  },
+  ApprovalController: {
+    approvalFlows: false,
+    pendingApprovals: false,
+    pendingApprovalCount: false,
   },
   AppStateController: {
+    browserEnvironment: true,
     connectedStatusPopoverHasBeenShown: true,
+    currentPopupId: false,
     defaultHomeActiveTabName: true,
+    fullScreenGasPollTokens: true,
+    hadAdvancedGasFeesSetPriorToMigration92_3: true,
+    nftsDetectionNoticeDismissed: true,
+    nftsDropdownState: true,
+    notificationGasPollTokens: true,
+    outdatedBrowserWarningLastShown: true,
+    popupGasPollTokens: true,
+    qrHardware: true,
+    recoveryPhraseReminderHasBeenShown: true,
+    recoveryPhraseReminderLastShown: true,
+    serviceWorkerLastActiveTime: true,
+    showBetaHeader: true,
+    showProductTour: true,
+    showTestnetMessageInDropdown: true,
+    snapsInstallPrivacyWarningShown: true,
+    termsOfUseLastAgreed: true,
+    timeoutMinutes: true,
+    trezorModel: true,
+    usedNetworks: true,
+  },
+  CachedBalancesController: {
+    cachedBalances: false,
+  },
+  CronjobController: {
+    jobs: false,
   },
   CurrencyController: {
     conversionDate: true,
     conversionRate: true,
     currentCurrency: true,
     nativeCurrency: true,
+    pendingCurrentCurrency: true,
+    pendingNativeCurrency: true,
+    usdConversionRate: true,
   },
   DecryptMessageController: {
+    unapprovedDecryptMsgs: false,
     unapprovedDecryptMsgCount: true,
   },
-  DesktopController: {
-    desktopEnabled: true,
-  },
   EncryptionPublicKeyController: {
+    unapprovedEncryptionPublicKeyMsgs: false,
     unapprovedEncryptionPublicKeyMsgCount: true,
   },
+  EnsController: {
+    ensResolutionsByAddress: false,
+  },
+  GasFeeController: {
+    estimatedGasFeeTimeBounds: true,
+    gasEstimateType: true,
+    gasFeeEstimates: true,
+  },
   KeyringController: {
     isUnlocked: true,
+    keyrings: false,
+    keyringTypes: false,
   },
   MetaMetricsController: {
+    eventsBeforeMetricsOptIn: false,
+    fragments: false,
     metaMetricsId: true,
     participateInMetaMetrics: true,
+    previousUserTraits: false,
+    segmentApiCalls: false,
+    traits: false,
   },
   NetworkController: {
+    networkConfigurations: false,
     networkId: true,
-    networkStatus: true,
+    networksMetadata: true,
     providerConfig: {
+      chainId: false,
```

Comment:
Maybe this should be included? We're a bit inconsistent here, including chain IDs in some cases but not others. We can always re-evaluate later though.

Reply:
```suggestion
      chainId: true,
```